api/controllers/FileUploadController.js

```diff
@@ -23,6 +23,12 @@ const FileUploadController = {
         const connectedToClamAV = await connectToClamAV(req);
         // @ts-ignore
         const userData = HelperService.getUserData(req, res);
+        const noFileUploadedError = Boolean(
+            req.flash('noFileUploadedError')[0]
+        );
+        const fileCountError = Boolean(req.flash('fileCountError')[0]);
+        const errors = req.flash('errors');
+        const infectedFiles = req.flash('infectedFiles');
```

Comment:
Putting the flash messages in variables seemed to retain them for me. Without it they'd disappear and not show on the page. Unless I call flash directly form the .ejs page.